src/frogger.cpp

```diff
@@ -113,21 +116,37 @@ Rcpp::List run_base_model(const Rcpp::List data,
   const leapfrog::TensorMap3<double> cd4_initdist(REAL(projection_parameters["cd4_initdist_full"]), disease_stages,
                                                   age_groups_hiv, num_genders);
   const leapfrog::TensorMap1<int> hiv_age_groups_span(INTEGER(projection_parameters["hAG_SPAN_full"]), age_groups_hiv);
+  const leapfrog::TensorMap4<double> art_mortality(REAL(projection_parameters["art_mort_full"]), treatment_stages,
+                                                   disease_stages, age_groups_hiv, num_genders);
+  const leapfrog::TensorMap2<double> artmx_timerr(REAL(projection_parameters["artmx_timerr"]), treatment_stages,
+                                                  proj_years);
+  leapfrog::Tensor1<double> h_art_stage_dur(treatment_stages - 1);
+  for (int i = 0; i < treatment_stages - 1; ++i) {
+    h_art_stage_dur(i) = 0.5;
+  }
+  const leapfrog::TensorMap1<double> art_dropout(REAL(projection_parameters["art_dropout"]), proj_years);
+  const leapfrog::TensorMap2<double> art15plus_num(REAL(projection_parameters["art15plus_num"]), num_genders,
+                                                   proj_years);
+  const leapfrog::TensorMap2<int> art15plus_isperc(INTEGER(projection_parameters["art15plus_isperc"]), num_genders,
+                                                   proj_years);
 
   const leapfrog::Parameters<double> params = {num_genders,
```

Comment:
this is going to become a source of bugs and pain in refactoring, and splitting into related chunks will likely help (later)